CMakeLists.txt

```diff
@@ -322,7 +322,7 @@ message("  Documentation ........ ${URIPARSER_BUILD_DOCS}")
 message("")
 message("Continue with")
 message("  make")
```

Comment:
I would remove that, CMake is not meant to be used with make only. By default it generates Makefiles on unixes but XCode on macOS, Visual Studio on Windows...

Reply:
Good point, I have limited the make part of that output to generator `Unix Makefiles` now.